sortfilterproxymodel.qbs

```diff
@@ -59,6 +61,7 @@ Product
         [
             "filter.cpp",
             "qqmlsortfilterproxymodel.cpp",
+            "qqmlsortfilterproxymodel_plugin.cpp",
```

Comment:
I'm fine with this staying like this if you'd prefer, as this is a third-party library.